### PR TITLE
Make sure `request` command does not crash for unhandled errors in `h…

### PR DESCRIPTION
**Pull request by kamalaknn**
…ttp-request` command.

### CODE REVIEW

File: lib/runner/extensions/request.command.js [hunk 1]
```diff
@@ -13,17 +13,20 @@ module.exports = {
...
-                        nextPayload.request,
-                        // this `item` is resolved and mutated here
-                        nextPayload.item,
-                        nextPayload.cookies
-                    );
+                    // nextPayload will be empty for unhandled errors
+                    if (nextPayload) {
+                        // trigger `response` callback
+                        // nextPayload.response will be empty for error flows
+                        this.triggers.response(
```
Comment (kunagpal) [inline lib/runner/extensions/request.command.js]: The `.response` call here could take `nextPayload` directly. This looks verbose.

Comment (kamalaknn) [reply lib/runner/extensions/request.command.js]: `triggers.response` is the public `response` callback. This the signature of it.

Comment (kunagpal) [reply lib/runner/extensions/request.command.js]: Fine for now, but this could be looked into later.

**Comment by shamasis** [comment]
LGTM + I committed one change myself @kamalaknn

**Comment by shamasis** [comment]
@kamalaknn - build failing at:

```
  35 passing (31s)
  1 failing
  1) Localhost requests
       IPv6 server
         "before each" hook: Start IPv6 server:
     Uncaught Error: listen EADDRNOTAVAIL ::1
      at Object.exports._errnoException (util.js:907:11)
      at exports._exceptionWithHostPort (util.js:930:20)
      at Server._listen2 (net.js:1240:19)
      at listen (net.js:1289:10)
      at net.js:1398:9
```

**Pull request merged**